doc/source/ray-core/gotchas.rst

```diff
@@ -111,18 +111,18 @@ of Ray Tasks itself, e.g.
   tuner = tune.Tuner(objective, param_space={"a": 1})
   tuner.fit()
 
-This will error with message: 
-ValueError: Cannot schedule create_task_that_uses_resources.<locals>.sample_task with the placement group 
+This will error with message:
+ValueError: Cannot schedule create_task_that_uses_resources.<locals>.sample_task with the placement group
 because the resource request {'CPU': 10} cannot fit into any bundles for the placement group, [{'CPU': 1.0}].
 
 **Expected behavior**: The above executes.
 
 **Fix**: In the ``@ray.remote`` declaration of tasks
 called by ``create_task_that_uses_resources()`` , include a
-``placement_group=None``.
+``scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group=None)``.
```

Comment:
Still feel a bit weird that we specify a strategy just to not use that strategy.

Reply:
We allow both right? Because in Datasets, we override using DEFAULT.

Reply:
Yes, currently we allow both.